metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -109,10 +128,35 @@ def remove_quotes(cls, v):
         v = v.replace('"', "").replace("`", "")
         return v
 
+    def preprocess_file_path(self, file_path: str) -> str:
+        new_file_path: str = str(file_path)
+
+        str_to_remove: List[str] = [
+            ".view.lkml",
+        ]
+
+        str_to_replace: Dict[str, str] = {
+            f"imported_projects/{self.project_name}/": "",
+            "/": "_",  # / is not urn friendly
```

Comment:
1. I personally prefer `.` over `_` for readability. Also `_` are common in folder names. Might lead to urn conflicts. e.g. if folder names are like `a_b / a_c.view.lkml `and `a / b_a / c.view.lkml`
2. Need accuracy check for imported_projects handling.
@hsheth2 - can you please review and confirm ?

Reply:
done